src/osd/osd_types.cc

```diff
@@ -1500,7 +1509,7 @@ void pg_pool_t::encode(bufferlist& bl, uint64_t features) const
     return;
   }
 
-  uint8_t v = 25;
+  uint8_t v = 26;
```

Comment:
fwiw that weird condition should instead be written if (!HAVE_FEATURE(features, SERVER_LUMINOUS))